Stock-signalling.py

```diff
@@ -0,0 +1,57 @@
+import requests
+
+
+# List ofstock symbols that will be monitored
+stock_symbols = ["TSLA", "AAPL", "MSFT", "GOOGL", "NKE"]
+

```

Comment:
I liked that you have blank lines to separate the different sections in your code. 
For consistency, I suggest choosing either one or two blank lines and applying it across the whole code. 
For example - there is one blank line separating the codes here, but in line 2 and 3 there are two lines, for the majority of the code there is one blank line, but in line 31 and 32 there are two blank lines.